config/application.rb

```diff
@@ -8,11 +8,13 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "sprockets/railtie"
+require 'carrierwave'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.